helix-core/src/main/java/org/apache/helix/manager/zk/zookeeper/IZkStateListener.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2010 the original author or authors.
```

Comment:
Nit: How come this is different from the following?

```
/*
 * Licensed to the Apache Software Foundation (ASF) under one
 * or more contributor license agreements.  See the NOTICE file
 * distributed with this work for additional information
 * regarding copyright ownership.  The ASF licenses this file
 * to you under the Apache License, Version 2.0 (the
 * "License"); you may not use this file except in compliance
 * with the License.  You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */
```

Reply:
Well, this interface and code is copied from I0Itec. I think we should keep the its original license, as what we do for ZkConnection. What do you think?